docs/BasicCommunication/GetSystemTime/index.md

```diff
@@ -0,0 +1,109 @@
+## GeSystemTime
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Obtain current UTC time  
+
+When an application  wants to open protected service it sends certificate to SDL during TLS or DTLS handshake.  
+In order to validate the certificate and to check whether this certificate is not expired SDL needs system time.
```

Comment:
In order to validate the certificate and to check whether this certificate is not expired SDL needs system time.
should be
In order to validate the certificate and to check whether this certificate is not expired SDL core needs an accurate UTC system time value.

Reply:
@mrapitis, thank you.
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/3c7a300842699307e9fd06ffb64fcd65d66d6d77